USAGE.md

```diff
@@ -1,3 +1,10 @@
+---
+layout: documentation
+title: Google Assistant
```

Comment:
"Google Assistant _what_"

Reply:
I just thought about really adding a `what` now, just for the sake of fun. 😄 

Is it called *skill* too there?
Or should we choose *integration*?

Reply:
😄 sorry

Hmm it's called "Action" in the Google Assistant world. https://assistant.google.com/services/a/uid/000000f5c61c627e?hl=en-US
I would like integration better. @marziman wdyt?

Reply:
+1 for integration

Lets wait for feedback.